cmd/root.go

```diff
@@ -142,3 +143,11 @@ func sendMessageToURL(msg string) bool {
 
 	return true
 }
+
+// encodeURL function to encode input endpoint URL
+func encodeURL(endpoint string) string {
+	parsedURL, _ := url.Parse(endpoint)
+	parsedQuery, _ := url.ParseQuery(parsedURL.RawQuery)
+	parsedURL.RawQuery = parsedQuery.Encode()
+	return parsedURL.String()
```

Comment:
@asdivekar Something's not right here. It's returning a URL without query params.